packages/@react-aria/selection/test/useSelectableCollection.test.js

```diff
@@ -107,5 +107,27 @@ describe('useSelectableCollection', () => {
       expect(options[1]).not.toHaveAttribute('aria-selected');
       expect(options[2]).toHaveAttribute('aria-selected', 'true');
     });
+
+    it("doesn't change the selection on focus in multiple selection selectOnFocus", async () => {
```

Comment:
We should get a test for `tab({shift: true});` as well

Also, expected behavior from Mac Finder suggests that clicking on any of the selected items while focus is currently inside the collection replaces the current selection.

However, if you leave the window (focus is no longer in the collection) and then the user clicks back in on any of the selected items, none of them are replaced.

We should have some tests around those behaviors as well and we'll need to decide what the overall behavior we ultimately want is. I'll bring it up to the team.

Reply:
Thanks! Good point about shift+tab, I can update the tests to cover it.

Not sure if "clicking" items should be considered relevant here. And it's handled separately.
What seems clear to me is that if there are selected items, the first/last one should get focused when tabbing into the collection without any side effect on the selection.